src/alexapi/device_platforms/raspberrypi.py

```diff
@@ -64,7 +64,7 @@ def indicate_recording(self, state=True):
 	def indicate_playback(self, state=True):
 		GPIO.output(self.__pconfig['plb_light'], GPIO.HIGH if state else GPIO.LOW)
 
-	def detect_button(self, channel):
```

Comment:
We actually need this. It's not used anywhere, but this is how the underlying C library calls it. It crashes if it's not there.